source/premiers-pas/deployer/installation.rst

```diff
@@ -43,3 +43,43 @@ L'installation à proprement parler de l'application consiste en trois étapes :
 #. :doc:`Lancer l'assistant d'installation en ligne <assistant-installation>`.
 
 Une fois ces étapes effectuées l'application est prête à être utilisée.
+
+Emplacement des fichiers et dossiers
+------------------------------------
+
+Comme beaucoup d'applications web, GLPI peut être installé en copiant simplement le contenu complet du répertoire sur un serveur web quelconque. Néanmoins, une telle pratique peut s'avérer relativement peu sécure.
+
+.. warning::
+
+   Tout fichier accessible directement depuis un serveur web doit être considéré comme insécure !
+
+En effet, GLPI stocke un certain nombre de données dans son répertoire ``files``, la configuration de votre base de données est stockée dans le dossier ``config``, ... Bien que des dispositions soient mises en place pour éviter que des données sensibles puisse être rendues accessibles directement depuis un serveur web ; la meilleure façon de les protéger reste de ne pas les rendre accessibles du tout.
+
+GLPI propose un certain nombre de variables de configuration qu'il est possible d'utiliser pour ce faire (utilisées d'ailleurs dans les paquets fournis par les distributions linux) :
+
+* ``GLPI_CONFIG_DIR`` : détermine le chemin vers le dossier de configuration ;
+* ``GLPI_VAR_DIR`` : détermine le chemin vers les "fichiers" ;
+* ``GLPI_LOG_DIR`` : détermine le chemin vers les fichiers de log.
+
+.. note::
+
+   Il existe bien d'autres variables de configuration, celles évoquées ci-dessus sont les principales à prednre en considération pour une installation davantage sécurisée.
+
+Le choix des différents dossiers est laissé à votre entière appréciation ; l'exemple qui suit se base quand à lui sur les recommandations `FHS <http://www.pathname.com/fhs/>`_.
+
+Notre instance de GLPI sera installée dans le dossier ``/var/www/glpi``, un hôte virtuel dans la configuration du serveur web reflètera ce chemin.
+
+Les données de GLPI seront stockées dans ``/var/lib/glpi/``, copier le contenu du dossier ``files`` à cet endroit. L'application devra posséder les droits en lecture et écriture sur ce dossier.
+
+La configuration de GLPI sera stockée dans ``/etc/glpi/``, copier le contenu du dossier ``config`` à cet endroit. L'application devra posséder les droits en lecture sur ce dossier, et en écriture également, mais uniquement pour l'installation.
+
+Les fichiers de log de GLPI seront stockés dans ``/var/log/glpi/``, il n'y a rien à copier. L'application devra posséder les droits en lecture et écriture sur ce dossier.
+
+Suivant cette nomenclature, créer un fichier ``inc/downstream.php`` avec le contenu suivant :
```

Comment:
Plutôt le fichier local_define.php ?

Reply:
C'est ce que j'avais mis en premier ; mais il ne permet pas de déplacer le dossier de configuration :/

Ce serait compliqué à expliquer/comprendre je pense. Du coup, je me suis dit que soit on utilisait downsteam.php ; soit on ne paramétrait pas le dossier de config (ce qui serait dommage à mon avis).

Reply:
> il ne permet pas de déplacer le dossier de configuration

Exact

Je pense qu'il faut donc parler des 2 (un simple ou) car dans le cas du packaging, downstream est utilisé et ne peut pas être modifié

Reply:
Comment in top of the inc/downstream.php file
```
/**
 * Red Hat / Fedora RPM default configuration
 *
 * Modifying this file in-place is not recommended, because
 * changes will be overwritten during package upgrades.
 *
 * If you want to customize the behaviour, the best way is to
 * create and use the /etc/glpi/local_define.php file.
 *
**/
```

Reply:
Oui, clairement... Je vais voir à être un peu plus précis.